### PR TITLE
First pass at docs for Google.Devtools.AspNet

### PR DESCRIPTION
**Pull request by jskeet**
This is just enough to get started, basically.
Also adjusted the example in the class itself to compile.

### CODE REVIEW

File: apis/Google.Devtools.AspNet/docs/index.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
+# Google.Devtools.AspNet
+
+`Google.Devtools.AspNet` is a ASP.NET instrumentation library for Google Stackdriver.
```
Comment (evildour) [inline apis/Google.Devtools.AspNet/docs/index.md]: a => an

Comment (jskeet) [inline apis/Google.Devtools.AspNet/docs/index.md]: Fixed, thanks.

**Comment by evildour** [comment]
LGTM, but probably best to have @iantalarico review as well.

**Comment by jskeet** [comment]
Just realized I missed creating a link in the root index.md file... will do that separately, then rebuild the docs and push.

**Comment by jskeet** [comment]
Fixed that, built the docs and pushed. Look ma, it's live: 
https://googlecloudplatform.github.io/google-cloud-dotnet/docs/Google.Devtools.AspNet/index.html

(That demonstrates that the example in CloudErrorReportingExceptionLogger needs fixing to be code, aside from anything else...)

**Pull request merged**